openid-connect-4-credential-issuance-1_0.md

```diff
@@ -476,8 +510,8 @@ OPTIONAL. Used when making a Signed Claimset Request, defines the key material t
 
 When the Issuer can immideately issue a requested credential and wants to send an issued credential to the Client, the Credential Response MUST return the following parameter:
 
-* credential
-  * REQUIRED. An Array of issued credentials.
+* `credential`: REQUIRED. the issued credentials.
```

Comment:
Credential property is a list
Do we expect endpoint returns multiple VCs or one-by-one?

Reply:
no